packages/analytics/src/api.ts

```diff
@@ -734,3 +736,21 @@ export function logEvent(
  * @public
  */
 export type CustomEventName<T> = T extends EventNameString ? never : T;
+
+/**
+ * Sets the applicable end user consent state for this web app across all gtag references once
+ * Firebase Analytics is initialized.
+ *
+ * Use the {@link ConsentSettings} to specify individual consent type values. By default consent
+ * types are set to "granted".
+ * @public
+ * @param consentSettings Maps the applicable end user consent state for gtag.js.
```

Comment:
Oops, I just ran `build` on this branch and noticed warnings from api-documenter that `consentSettings` needs to be followed by a hyphen here, and also customParams on line 236, which I think was from a previous PR.